src/components/atoms/Image.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
     export let src
```

Comment:
Why only one is initialized?

Reply:
Actually, same answer that above haha. But as this one was written for me I can say that src prop should not be initialized by default. It is a mandatory prop for the image component.